### PR TITLE
Regenerate hammer plugins based on latest template

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/plugins/rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec [hunk 1]
```diff
@@ -1,70 +1,74 @@
...
+
+Name: rubygem-%{gem_name}
 Version: 0.1.13
-Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
-License: GPLv3
-URL: https://github.com/theforeman/hammer_cli_foreman_openscap
+Release: 2%{?foremandist}%{?dist}
+Summary: Foreman OpenSCAP commands for Hammer
+License: GPL-3
```
Comment (evgeni) [inline packages/plugins/rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec:11]: `GPL-3` doesn't look correct?

Comment (ekohl) [reply packages/plugins/rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec]: Are you saying it should be `GPL-3.0` upstream to be valid SPDX? Then our [license.json](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/licenses.json) will also understand it.

Comment (evgeni) [reply packages/plugins/rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec]: That's what I said and committed to :)

Comment (ekohl) [reply packages/plugins/rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec]: Well, GH didn't show your message until I submitted. I blame that.

File: packages/plugins/rubygem-hammer_cli_foreman_ssh/rubygem-hammer_cli_foreman_ssh.spec [hunk 2]
```diff
@@ -1,78 +1,72 @@
...
+%{gem_spec}
+%config(noreplace) %{hammer_confdir}/cli.modules.d/%{plugin_name}.yml
 
 %files doc
 %doc %{gem_docdir}
 %doc %{gem_instdir}/README.md
+%doc %{gem_instdir}/config
 
 %changelog
+* Wed Oct 19 2022 Evgeni Golov 0.0.3-1
```
Comment (ekohl) [inline packages/plugins/rubygem-hammer_cli_foreman_ssh/rubygem-hammer_cli_foreman_ssh.spec:67]: It only came out in 2015. Don't you think it's too bleeding edge?

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Looks like it now requires `net-ssh-multi`:
```
[2022-10-19T10:50:46.972Z]     package: rubygem-hammer_cli_foreman_ssh-0.0.3-1.fm3_5.el8.noarch from downloaded_rpms
[2022-10-19T10:50:46.972Z]       unresolved deps:
[2022-10-19T10:50:46.972Z]         rubygem(net-ssh-multi) >= 1.2.1
```

**Comment by evgeni** [comment]
Always did?

https://github.com/theforeman/hammer-cli-foreman-ssh/blob/52b0de73a46a4de757d2a1e8b20555e664be2dd0/hammer_cli_foreman_ssh.gemspec#L17

**Comment by ekohl** [comment]
Hmm you're right. However, we only package 1.2.0, not 1.2.1.

**Comment by evgeni** [comment]
> Hmm you're right. However, we only package 1.2.0, not 1.2.1.

Nothing as easy as that ;)

File: packages/plugins/rubygem-net-ssh-multi/rubygem-net-ssh-multi.spec [hunk 3]
```diff
@@ -1,91 +1,69 @@
...
+Release: 1%{?dist}
 Summary: Control multiple Net::SSH connections via a single interface
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.2.0
-Release: 10%{?dist}
-Group: Development/Languages
 License: MIT
 URL: https://github.com/net-ssh/net-ssh-multi
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-Patch0: rubygem-net-ssh-multi-1.2.0-minitest.patch
```
Comment (evgeni) [inline packages/plugins/rubygem-net-ssh-multi/rubygem-net-ssh-multi.spec:17]: the patch is part of 1.2.1, so it's OK that it's gone

File: packages/plugins/rubygem-net-ssh-multi/rubygem-net-ssh-multi.spec [hunk 4]
```diff
@@ -1,91 +1,69 @@
...
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
-%check
-%if 0%{?enable_check}
```
Comment (evgeni) [inline packages/plugins/rubygem-net-ssh-multi/rubygem-net-ssh-multi.spec:64]: we had enable check hardcoded to 0 anyways, so :woman_shrugging:

**Review by ekohl — APPROVED** [review]

**Pull request merged**